includes/wikia/services/UserStatsService.class.php

```diff
@@ -134,30 +131,44 @@ function () use ( $flags ) {
 					$stats['lastContributionTimestamp'] = $this->initLastContributionTimestamp();
 				}
 
+				if ( $stats->needsUpdate() ) {
+					$this->scheduleStatsUpdateTask( $stats );
```

Comment:
It is an unfortunate side-effect of how we store edit count information. When loading edit count for user from DB, some of the data may not yet be initialized, so they have to be calculated on-demand and persisted to DB.